skeletor/common/css.config.js

```diff
@@ -13,7 +13,8 @@ module.exports = {
     	require('postcss-custom-selectors'),
     	require('postcss-custom-media'),
    		require('postcss-color-mod-function'),
-    	require('postcss-nesting'),
+		require('postcss-nesting'),
+		require('postcss-url'),
```

Comment:
[fix] The postcss-url plugin can accept a `url` option that we should set up here, too. I can send you an example.

Reply:
Done